amgut/db/patches/0034.sql

```diff
@@ -0,0 +1,31 @@
+--Feb 9, 2016
+--Add environmental sample survey
+
+INSERT INTO survey_group (group_order, american, british) VALUES (-3, 'Environmental Information', 'Environmental Information');
+
+----------------------------------------------------------
+-- surveys
+----------------------------------------------------------
+INSERT INTO surveys (survey_id, survey_group) VALUES (3, -3);
+
+----------------------------------------------------------
+-- survey_question
+----------------------------------------------------------
+INSERT INTO survey_question (survey_question_id, question_shortname, american, british) VALUES (162, 'ENV_NAME', 'Descriptive name for the sample', 'Descriptive name for the sample');
+INSERT INTO survey_question (survey_question_id, question_shortname, american, british) VALUES (163, 'ZIP', 'Zipcode sample was collected in', 'Zipcode sample was collected in');
```

Comment:
Ah, should be postcode, but yeah, international waters is...interesting. Could also ask for lat/long directly and use that instead if given.